lighthouse-core/gather/gatherers/image-elements.js

```diff
@@ -193,6 +264,25 @@ class ImageElements extends Gatherer {
     }
   }
 
+  /**
+   * @param {Driver} driver
+   * @param {string} devtoolsNodePath
+   * @param {LH.Artifacts.ImageElement} element
+   */
+  async fetchSourceRules(driver, devtoolsNodePath, element) {
+    const {nodeId} = await driver.sendCommand('DOM.pushNodeByPathToFrontend', {
```

Comment:
we'll want to wrap these protocol commands in try/catch in case some elements got removed in the interim. we see this sometimes with carousels and we don't want the whole gatherer to fail if one image couldn't get css data

example: https://github.com/GoogleChrome/lighthouse/blob/46b0b88778271327ea8ed1bcbb2c568addc8a1e8/lighthouse-core/gather/driver.js#L1237-L1245

Reply:
maybe we should have a `driver.trySendCommand` fn

Reply:
I would so be on board with that :) 

signature of something like `{result?: T, error?: Error, suceess: boolean}`? could handle automatically warning and stuff too if we wanted

a separate discussion though @lemcardenas I think we can proceed with try/catch for this immediate moment